README.md

```diff
@@ -1,50 +1,115 @@
 # Blindnet Java SDK
+<h1 align="center">
+  blindnet devkit<br />
+  Java Client SDK
+</h1>
 
-This repository contains Java implementation of Blindnet SDK.
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163277439-edd00509-1d1b-4565-a0d3-49057ebeb92a.png#gh-light-mode-only" height="80" /></p>
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163549893-117bbd70-b81a-47fd-8e1f-844911e48d68.png#gh-dark-mode-only" height="80" /></p>
 
-The SDK consists of two submodules. The Core submodule and the Signal submodule.
-The implementation of these submodules resides in the packages named same as the submodules.
+<p align="center">
+  <strong>{One-Liner: Explain concretely what this project does in one short sentence}</strong>
+</p>
 
-The project uses Gradle build tool.
+<p align="center">
+  <a href="https://blindnet.dev"><strong>blindnet.dev</strong></a>
+</p>
 
-# Requirements
+<p align="center">
+  <a href="https://blindnet.dev/docs">Documentation</a>
+  &nbsp;•&nbsp;
+  <a href="https://github.com/blindnet-io/{project-short-name}/issues">Submit an Issue</a>
+  &nbsp;•&nbsp;
+  <a href="https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg">Online Chat</a>
+  <br>
+  <br>
+</p>
 
-```
-Java v11+
-Gradle v7.3.1+
-```
+## About
 
-# Building project
+{explain what this project does and why it exists in the most concise way possible.}
 
-In order to build the project run:
-```
-gradle build
-```
+Those functionnalities are provided through two submodules:
 
-In order to run tests:
-```
-gradle clean test
+1. Core: {short explainer of what this module provide}
+2. Signal: {short explainer of what this module provide}
+
+## Get Started
+
+:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.
+
+## Installation
+
+This project requires [Java v11+](URL) {...} [Gradle v7.3.1+](URL) {...}
+
+Use [{package manager or plateform}][install-tool] to install {project's name}:
+
+```bash
+{install-command}
 ```
 
-# Using SDK
+## Usage
+
+<!-- FIXME: RESTRUCTURE and REWRITE in plain and correct english, using clear code examples -->
+
+<!-- FIXME: publish the API reference -->
+
+> 📑 The API reference of blindnet devkit Java client SDK is available on [blindnet.dev](https://blindnet.dev/docs/api_reference/[path-to-project}/latest).
```

Comment:
TODO @blindnet-io/engineering

Reply:
@veselinovicst you can comment this line, we'll address this in another issue and PR (see https://github.com/blindnet-io/sdk-java-server/pull/2#discussion_r929941909).

Reply:
```suggestion
```